src/libs/ErrorUtils.ts

```diff
@@ -197,6 +205,25 @@ function isReceiptError(message: unknown): message is ReceiptError {
     return ((message as Record<string, unknown>)?.error ?? '') === CONST.IOU.RECEIPT_ERROR;
 }
 
+/**
+ * Check if the error includes a translation key.
+ */
+function isTranslationKeyError(message: unknown): message is TranslationKeyError {
+    if (!message) {
+        return false;
+    }
+    if (typeof message === 'string') {
+        return false;
+    }
+    if (Array.isArray(message)) {
+        return false;
+    }
```

Comment:
Could be nice to cut the number of lines down here, what do y'all think?

```suggestion
    if (!message || typeof message === 'string' || Array.isArray(message)) {
        return false;
    }
```